Makefile

```diff
@@ -374,6 +380,20 @@ unit-test: envtest
 	OPERATOR_NAMESPACE=$(OPERATOR_NAMESPACE) KUBEBUILDER_ASSETS="$(shell $(ENVTEST) use $(ENVTEST_K8S_VERSION) --bin-dir $(LOCALBIN) -p path)" go test $(TEST_SRC) -v  -coverprofile cover.out
 CLEANFILES += cover.out
 
+.PHONY: extract-alert-rules
+extract-alert-rules: $(PROMETHEUS_CONFIG_YAML) 
+	./tests/prometheus_unit_tests/scripts/extract_alerts.sh $(PROMETHEUS_CONFIG_YAML) $(GENERATED_ALERT_DIR)
+
+# Run prometheus-alert-unit-tests
+.PHONY: test-alerts
+test-alerts: extract-alert-rules
+	cd "$(GENERATED_ALERT_DIR)" && promtool test rules *_unit_tests.yaml
```

Comment:
Does it require `cd` or giving it `$(GENERATED_ALERT_DIR)/*_unit_tests.yaml` works as well? Or there are other reasons for it?

Reply:
`$(GENERATED_ALERT_DIR)/*_unit_tests.yaml` works as well, just something I missed. Updating that now. Thanks!